### PR TITLE
Add Accept-Encoding header to cache key.

### PR DESCRIPTION
**Pull request by sersut**

### CODE REVIEW

**Comment by chef-delivery** [comment]
This PR has passed 'Verify' and is ready for review and approval!
  Use: '@delivery approve' when code review is complete.

**Comment by schisamo** [comment]
:+1:

**Comment by sersut** [comment]
@delivery approve

**Comment by chef-delivery** [comment]
Change: 21ac2d35-1c06-4648-8dd9-52c490f77a60 approved by: @sersut

File: .delivery/build-cookbook/recipes/provision.rb [hunk 1]
```diff
@@ -181,7 +181,7 @@
 fastly_request_setting 'cache_key' do
   api_key fastly_creds['api_key']
   service fastly_service.name
-  hash_keys 'req.url, req.http.host, req.http.Fastly-SSL, req.http.accept'
+  hash_keys 'req.url, req.http.host, req.http.Fastly-SSL, req.http.accept req.http.Accept-Encoding'
```
Comment (cwebberOps) [inline .delivery/build-cookbook/recipes/provision.rb]: You are missing a comma. Should be `req.http.accept, req.http.Accept-Encoding`

**Comment by sersut** [comment]
argh got it... thanks :)

**Pull request merged**